lang/extract_json_strings.py

```diff
@@ -21,7 +21,62 @@ def __str__(self):
     "LOADING_ORDER.md"
 }
 
-# don't parse this files. Full related path.
+# these are official mods need to be localized
+# list could be generated by bash command: (in data/mods directory)
+# for i in $(ls -l1); do echo -n '    "'; echo -n $i; echo '",'; done
+# and remove things like replacements.json manually
+official_mods = {
+    "Animatronics",
```

Comment:
Yes, this is a problem. What if add an `official` flag in those official provided mods?